Demo Scripts/scr_introduction_answer_fail.md

```diff
@@ -0,0 +1,40 @@
+## scr_introduction_answer_fail()
+
+No arguments.
+
+### Returns: N/A
+### Example:
+```gml
+if (jumped == true) // just an example, not real code
+    scr_introduction_answer_fail()
+```
+This function plays a voiceline depending on the times the action has been repeated. It adds to the "reset counter" in the intro and changes the voiceline every time until the last time.
+
+All the voiceline code:
```

Comment:
Please refer to CONTRIBUTING.md, please *never* include unmodified code from the game, especially large chunks like this. Our jobs are to describe a function's use and usage, not it's contents, modders can check that themselves.